README.md

```diff
@@ -3,73 +3,9 @@
 
 # Evidential<a name="evidential"></a>
 
-<!-- mdformat-toc start --slug=github --maxlevel=2 --minlevel=1 -->
-
-- [Evidential](#evidential)
-  - [Getting Started](#getting-started)
-  - [Documentation](#documentation)
-
-<!-- mdformat-toc end -->
-
 Evidential is an experiments management platform built on FastAPI, Postgres, and React.
```

Comment:
and we could have Evidential here link to the main docs https://docs.evidential.dev/